phys2bids/phys2bids.py

```diff
@@ -107,7 +158,10 @@ def use_heuristic(heur_file, sub, ses, filename, outdir):
     heur = utils.load_heuristic(heur_file)
     name = heur.heur(filename[:-4], name)
 
-    heurpath = fldr + '/' + name + '_physio'
+    if record_label:
+        recording = f'_recording-{record_label}'
+
+    heurpath = fldr + '/' + name + recording + '_physio'
```

Comment:
Hi guys, 

Leaving apart the "Good first issue" #40 apart. the variable ``recording`` is defined inside the if and therefore if not defined before as an empty string or something if ``record_label`` is left as default, the function will give and error for sure. I think either you put ``record_label`` as a required argument so there is no problem and then delete that if statement or define ``recording`` before the conditional statement.